README.md

```diff
@@ -8,6 +8,7 @@ It is officially maintained by the CoreOS team and meant to be a set of introduc
 
 When designing these guides and tools, the following considerations are made:
 
+* You must use a CoreOS version 773.1.0+ for the `kubelet` to be present in the image
```

Comment:
Instead maybe move this to a "prerequisites" section at top of getting started guides then (https://github.com/coreos/coreos-kubernetes/blob/master/Documentation/getting-started.md)? e.g. maybe prerequisites are: CoreOS version x.y.z, deployment options, generating tls certificates.